src/workload/mod.rs

```diff
@@ -510,6 +555,48 @@ impl Distribution {
 
 impl Keyspace {
     pub fn new(config: &Config, keyspace: &config::Keyspace) -> Self {
+        let value_random_bytes = if keyspace.compression_ratio() <= 1.0 || keyspace.vlen().is_none()
+        {
+            // this indicates the message should not be compressible, to achieve
+            // this all bytes will be random
+            keyspace.vlen().unwrap_or(0)
```

Comment:
I think we should hoist the option missing check outside and just set it to 0 if that is the case and only enter the `if..else` if it has been specified. This simplifies the logic, imo, but also makes it easier to use common code between topics and keyspace.

Reply:
Done.